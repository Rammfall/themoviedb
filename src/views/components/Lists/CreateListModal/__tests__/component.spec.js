import React from 'react'
import { shallow } from 'enzyme'

import CreateListModalComponent from '../component'

describe('CreateListModalComponent', () => {
  it('matches snapshot', () => {
    const props = {
      isVisible: false,
      toggleHandler: jest.fn,
      isSubmitting: false,
      handleSubmit: jest.fn
    }
    const component = shallow(<CreateListModalComponent {...props} />)

    expect(component).toMatchSnapshot()
  })
})
