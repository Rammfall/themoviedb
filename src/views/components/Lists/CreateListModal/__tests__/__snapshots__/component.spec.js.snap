// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CreateListModalComponent matches snapshot 1`] = `
<Modal
  confirmLoading={false}
  okButtonProps={
    Object {
      "htmlType": "submit",
    }
  }
  okText="Create"
  okType="primary"
  onCancel={[Function]}
  onOk={[Function]}
  title="Create list"
  visible={false}
  width={520}
>
  <Form>
    <Field
      component={[Function]}
      name="name"
      placeholder="Name"
    />
    <Field
      component={[Function]}
      name="description"
      placeholder="Description"
    />
  </Form>
</Modal>
`;
